pkg/lightning/backend/local.go

```diff
@@ -608,7 +608,6 @@ func (local *local) openEngineDB(engineUUID uuid.UUID, readOnly bool) (*pebble.D
 		L0CompactionThreshold: local.maxOpenFiles / 2,
 		L0StopWritesThreshold: local.maxOpenFiles / 2,
 		MaxOpenFiles:          local.maxOpenFiles,
-		DisableWAL:            true,
```

Comment:
Why remove this

Reply:
> "We enabled WAL to ensure a flushed Batch Write survives even if Lightning crashes."

it is in the PR description 😓

Reply:
I think the `WAL` is to ensure a commited batch can be recovered even if it's not flushed. Without WAL, a flushed batch can still survive even if lightning crashes?🤔

BTW I'd like to know the performance regression with WAL on and off. If it's non-trivial, I suspect is it worth we pay this price only to address the rare crash.